index.html

```diff
@@ -165,7 +165,51 @@ <h1 class="crete w400 s72 lh88 ls037 orange">About me</h1>
                 </li>
             </ul>
         </section>
-        
+        <section id="contact">
+            <p class="inter nBlue2 s32 w700 lh44">I'm always<br>interested in hearing<br>about new projects,<br>so if you'd 
+               like to<br>chat please get in touch.
+            </p>
+            <form action="https://formspree.io/f/xlevqbpl" method="post">
+                <ul>
+                    <li class="formName">
+                        <label for="name" class="inter w400 s15 lh24">Full name</label>
+                        <input type="text" id="name" name="user_name" maxlength="30" required />
+                    </li>
+                    <li class="formEmail">
```

Comment:
- [ ] I  think it would be better to remove the `ul` and `li` as it's already wrapped inside a `<form>` container .